### PR TITLE
Removed the dups and did some formattings

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

**Comment by chenrui333** [comment]
cc @markthink @zhangxiaoyu-zidif

**Comment by chenrui333** [comment]
@Rajakavitha1 how can I get approve for this PR?

File: content/en/docs/concepts/containers/runtime-class.md [hunk 1]
```diff
@@ -91,8 +88,8 @@ spec:
 {{< note >}}
 
 **Note:** It is recommended that RuntimeClass write operations (create/update/patch/delete) be
-restricted to the cluster administrator. This is typically the default. See [Authorization
-Overview](https://kubernetes.io/docs/reference/access-authn-authz/authorization/) for more details.
+restricted to the cluster administrator. This is typically the default.
+See [Authorization Overview](https://kubernetes.io/docs/reference/access-authn-authz/authorization/) for more details.
```
Comment (stewart-yu) [inline content/en/docs/concepts/containers/runtime-class.md]: should we need change this?

File: content/en/docs/concepts/containers/runtime-class.md [hunk 2]
```diff
@@ -38,13 +37,11 @@ the RuntimeClass feature:
 #### 1. Enable the RuntimeClass feature gate
 
 See [Feature Gates](/docs/reference/command-line-tools-reference/feature-gates/) for an explanation
-of enabling feature gates. The `RuntimeClass` feature gate must be enabled on apiservers _and_
-kubelets.
+of enabling feature gates. The RuntimeClass feature gate must be enabled on apiservers _and_ kubelets.
```
Comment (stewart-yu) [inline content/en/docs/concepts/containers/runtime-class.md]: why change?

File: content/en/docs/concepts/containers/runtime-class.md [hunk 3]
```diff
@@ -38,13 +37,11 @@ the RuntimeClass feature:
...
 
 #### 2. Install the RuntimeClass CRD
 
-The RuntimeClass [CustomResourceDefinition][/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/] (CRD) can be found in the addons directory of the
-Kubernetes git repo:
+The RuntimeClass [CustomResourceDefinition][/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/] (CRD) can be found in the addons directory of the Kubernetes git repo:
```
Comment (stewart-yu) [inline content/en/docs/concepts/containers/runtime-class.md]: no need?

File: content/en/docs/concepts/containers/runtime-class.md [hunk 4]
```diff
@@ -61,12 +58,12 @@ feature, not all CRIs support multiple RuntimeClasses yet.
 {{< note >}}
 **Note:** RuntimeClass currently assumes a homogeneous node configuration across the cluster
 (which means that all nodes are configured the same way with respect to container runtimes). Any heterogeneity (varying configurations) must be
-managed independently of RuntimeClass through scheduling features (see [Assigning Pods to
-Nodes](/docs/concepts/configuration/assign-pod-node/)).
+managed independently of RuntimeClass through scheduling features
+(see [Assigning Pods to Nodes](/docs/concepts/configuration/assign-pod-node/)).
 {{< /note >}}
```
Comment (stewart-yu) [inline content/en/docs/concepts/containers/runtime-class.md]: The same, no need

File: content/en/docs/concepts/containers/runtime-class.md [hunk 5]
```diff
@@ -121,127 +118,3 @@ If no `runtimeClassName` is specified, the default RuntimeHandler will be used,
...
-RuntimeClass does not exist, or the CRI cannot run the corresponding handler, the pod will enter the
-`Failed` terminal [phase](/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase). Look for a
-corresponding [event](/docs/tasks/debug-application-cluster/debug-application-introspection/) for an
-error message.
-
-If no `runtimeClassName` is specified, the default RuntimeHandler will be used, which is equivalent
-to the behavior when the RuntimeClass feature is disabled.
-
-{{% /capture %}}
->>>>>>> master
```
Comment (stewart-yu) [inline content/en/docs/concepts/containers/runtime-class.md]: good catch :+1:

File: content/en/docs/concepts/containers/runtime-class.md [hunk 6]
```diff
@@ -61,12 +58,12 @@ feature, not all CRIs support multiple RuntimeClasses yet.
...
 (which means that all nodes are configured the same way with respect to container runtimes). Any heterogeneity (varying configurations) must be
-managed independently of RuntimeClass through scheduling features (see [Assigning Pods to
-Nodes](/docs/concepts/configuration/assign-pod-node/)).
+managed independently of RuntimeClass through scheduling features
+(see [Assigning Pods to Nodes](/docs/concepts/configuration/assign-pod-node/)).
 {{< /note >}}
 
 The configurations have a corresponding `RuntimeHandler` name, referenced by the RuntimeClass. The
-RuntimeHandler must be a valid DNS 1123 subdomain (alpha-numeric + `-` and `.` characters).
+RuntimeHandler must be a valid DNS 1123 subdomain (alpha-numeric, `+`, `-`, and `.` characters).
```
Comment (stewart-yu) [inline content/en/docs/concepts/containers/runtime-class.md]: Change seems wrong, `a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.'`, so plz replace `+` with `,`.

Comment (chenrui333) [reply content/en/docs/concepts/containers/runtime-class.md]: oh, yeah, you are right. Can I update the line to be like:
```
valid DNS-1123 subdomain (alpha-numeric characters, `-` or `.`)
```

File: content/en/docs/concepts/containers/runtime-class.md (hunk 1 above)
Comment (chenrui333) [reply content/en/docs/concepts/containers/runtime-class.md]: This change is necessary if you have editor to do the syntax highlighting. I think it is needed.

**Comment by chenrui333** [comment]
@stewart-yu thanks for the feedback, I have made some updates, please give another look.

File: content/en/docs/concepts/containers/runtime-class.md (hunk 1 above)
Comment (stewart-yu) [reply content/en/docs/concepts/containers/runtime-class.md]: can't follow

File: content/en/docs/concepts/containers/runtime-class.md (hunk 4 above)
Comment (stewart-yu) [reply content/en/docs/concepts/containers/runtime-class.md]: can't follow

**Pull request merged**